ec2api/api/cloud.py

```diff
@@ -1339,6 +1340,30 @@ def update_quota(self, context, account, resource, quota):
     def show_quota(self, context, account):
         """
         """
+    
+    @module_and_param_types(account, 'account_id')
+    def create_paas_account(self, context, account_id):
+        """ Creates a PASS account entry for specified account
+        
+        Args:
+            context (RequestContext): The request context.
+            account_id : ID of the account for which PASS must be enabled
```

Comment:
Please add on what is the format of account-id . What is valid and what is invalid